src/providers/perplexity.ts

```diff
@@ -29,11 +43,13 @@ const Perplexity: Provider = {
 
     return {
       messageText: response.choices[0]?.message.content ?? null,
-      stats: {
-        responseTime,
-        inputTokens: response.usage?.prompt_tokens,
-        outputTokens: response.usage?.completion_tokens,
+      usage: {
+        inputTokens: response.usage?.prompt_tokens ?? 0,
+        outputTokens: response.usage?.completion_tokens ?? 0,
+        requests: 1,
```

Comment:
Does it make sense to add the request counter now, when every call is returning 1?

Reply:
`requests: 1` simplified summing up of usage in `combineUsage`